code-challenges.test.js

```diff
@@ -10,60 +10,155 @@
 // Reminder: The test will call your function
 // Run the file with the following command: $ yarn jest
 
-// --------------------INSTRUCTOR EXAMPLE: Create a function that takes in an array of numbers and returns an array with all the numbers multipled by 3.
+// leave test and something commented in
 
+// --------------------INSTRUCTOR EXAMPLE: Create a function that takes in an array of numbers and returns an array with all the numbers multipled by 3.
 // a) Create a test with expect statements for each of the variables provided.
 
+describe ("multBy3", () => {
+    it("takes in an array of numbers and returns an array with all the numbers multipled by 3", () => {
+     const numbersArray1 = [6, 7, 8, 9, 10]
+     const numbersArray2 = [24, 27, 30, 33, 36]
+      expect(multBy3(numbersArray1)).toEqual([18, 21, 24, 27, 30])
+      expect(multBy3(numbersArray2)).toEqual([72, 81, 90, 99, 108])
+    })
+  })
 const numbersArray1 = [6, 7, 8, 9, 10]
 // Expected output: [18, 21, 24, 27, 30]
 const numbersArray2 = [24, 27, 30, 33, 36]
 // Expected output: [72, 81, 90, 99, 108]
+// ReferenceError : multby3 in not defined 
```

Comment:
Your test looks great! Since you have the test variables in the test they don't need to be duplicated.